paddle/fluid/operators/distributed/request_handler.h

```diff
@@ -68,12 +68,14 @@ class VarHandle {
 
  public:
   bool Wait() {
+    int ret = kVarHandleDefaultState;
     {
       std::unique_lock<std::mutex> lk(sync_mutex_);
       wait_cond_.wait(lk, [this] { return ok_ != kVarHandleDefaultState; });
+      ret = ok_;
```

Comment:
wait is a blocking call, why need a return value?

Reply:
Done. Thanks!